cmd/login/login.go

```diff
@@ -72,20 +73,67 @@ func doLogin(accessToken string) error {
 	if err != nil {
 		return err
 	}
+	appid = int64(resObj.AppID)
 	creds, _ := json.Marshal(resObj)
 	err = cmdutil.WriteFile("token.json", creds)
 	if err != nil {
 		return err
 	}
 	fmt.Println("Successfully Authenticated, Fetching Your Site(s)...")
-	_, err = api.GetAppsInfo()
+	err = listSites()
 	if err != nil {
 		return err
 	}
 	fmt.Println("Successfully Logged In")
 	return nil
 }
 
+func listSites() error {
+	m := make(map[int]int64)
+	var option string
+	var siteChoice int
+	appInfo, err := api.GetAppsInfo()
+	if err != nil {
+		return err
+	}
+	var i int
+	fmt.Println("List of sites: ")
+	for ID, App := range appInfo {
+		i = i + 1
+		if appid == ID {
+			fmt.Println(i, "-", App.Appname, "(Default site)")
+		} else {
+			fmt.Println(i, "-", App.Appname)
+			m[i] = ID //store ID into map except for the default site
+		}
+	}
+	fmt.Printf("Do you wish to start with a different site ?(Y/N): ")
+	fmt.Scanf("%s\n", &option)
+	if option != "Y" {
+		return nil
+	}
+
+	fmt.Printf("Enter the corresponding number of the site as displayed above: ")
+	fmt.Scanf("%d\n", &siteChoice)
+	if siteChoice > len(appInfo) || siteChoice <= 0 {
+		fmt.Println("Invalid choice")
```

Comment:
Are we restricting the user after the invalid choice?, Ideally the message should be  something like this`Invalid Choice, Switching to your default site`.